packages/explorer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@latticexyz/explorer",
+  "version": "2.1.1",
+  "private": true,
+  "description": "Explorer is a tool for visually exploring and manipulating the state of worlds",
+  "bin": {
+    "explorer": "./dist/explorer.mjs"
+  },
+  "scripts": {
+    "build": "pnpm run build:explorer && pnpm run build:bin",
+    "build:bin": "tsup && pnpm link .",
```

Comment:
why link?

Reply:
ah, it was useful in the beginning for testing the `explorer` command but actually not really needed, removed